postman-generator/src/main/scala/examples/RandomStringGenerator.scala

```diff
@@ -0,0 +1,26 @@
+package examples
+
+import scala.util.Random
+import scala.util.hashing.MurmurHash3
+
+trait RandomStringGenerator {
+  def generate(seed: String): String
+}
+
+object ScalaRandomStringGenerator extends RandomStringGenerator {
+
+  override def generate(seed: String): String = {
+    val string = randomStringStream.take(6).mkString
+    s"lorem ipsum $string"
```

Comment:
I think we could change it to smth like `s"lorem-ipsum-$string"`, casue sometimes this string is going to be an ID and this will increase chances to not get a 400.

Reply:
agree